packages/quantic/force-app/main/default/lwc/quanticDateFacet/quanticDateFacet.html

```diff
@@ -22,18 +22,22 @@
           </button>
         </template>
         <template if:false={isCollapsed}>
-          <ul class="slds-has-dividers_around-space slds-m-top_medium">
-            <template for:each={values} for:item="v">
-              <c-quantic-facet-value
-                onselectvalue={onSelectValue} 
-                key={v.start} 
-                item={v} 
-                is-checked={v.checked}
-                formatting-function={formattingFunction}
-              >
-              </c-quantic-facet-value>
-            </template>
-          </ul>
+          <fieldset>
```

Comment:
That's the kind of change I was expecting, short and sweet and it resolves the accessibility issues wtihout touching anything else.